.github/workflows/main.yml

```diff
@@ -6,25 +6,31 @@ on:
       - '*'
     tags:
       - '*'
+
 jobs:
   snapshot:
     name: Snapshot build
     runs-on: ubuntu-latest
     if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
     steps:
+      # Checkout so the workflow can access the repository
       - name: Checkout
```

Comment:
The comments are quite redundant in my opinion, the `name` should be descriptive enough? (and if not it should be improved)

Reply:
We agree with you on this point, the name should be descriptive enough. We can propose you an update.